networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/AttestationGossipManager.java

```diff
@@ -60,7 +62,8 @@ public void onNewAttestation(final ValidateableAttestation validateableAttestati
               attestationPublishSuccessCounter.inc();
             },
             error -> {
-              LOG.trace(
+              LOG.log(
+                  error instanceof NoPeersForOutboundMessageException ? Level.DEBUG : Level.WARN,
```

Comment:
I think this is going to spam the logs when we run a single node network as it never has any peers. Probably not the end of the world but it does get annoying.

Reply:
Ah right sorry, in that case it would be debug which is fine.  I'd suggest other exceptions should be `ERROR` level then rather than `WARN` as I wouldn't expect those (unless there's some exception type I'm missing).

Reply:
I expect networking exceptions. changing to ERROR.